concent_api/verifier/tasks.py

```diff
@@ -179,6 +183,33 @@ def blender_verification_order(
     for file_path in source_files_list + [source_package_path]:
         delete_file(file_path)
 
+    blender_output_file_name = generate_blender_output_file_name(scene_file)
+    upload_file_name = generate_upload_file_name(subtask_id)
+    (file_size, file_check_sum) = get_file_size_and_checksum(blender_output_file_name)
+
+    try:
+        # Generate a FileTransferToken valid for an upload of the image generated by blender.
+        upload_file_transfer_token = create_file_transfer_token_for_concent(
+            subtask_id=subtask_id,
+            result_package_path=upload_file_name,
+            result_size=file_size,
+            result_package_hash=file_check_sum,
+            operation=message.FileTransferToken.Operation.upload,
+        )
+
+        # Upload the image.
+        with open(os.path.join(settings.VERIFIER_STORAGE_PATH, blender_output_file_name)) as f:
+            upload_file_to_storage_cluster(
+                f.read(),
+                upload_file_name,
+                upload_file_transfer_token,
+            )
+
+        # Delete the image from local storage.
+        delete_file(blender_output_file_name)
+    except OSError as e:
+        logger.info(f'Uploading blender generated file to storage fails with: {e}')
```

Comment:
This is an error not info. You should also log the exception. And report it to Sentry.

Reply:
:+1: done but please confirm that I have used it correctly.